### PR TITLE
Uninstall instructions

### PR DESCRIPTION
**Pull request by lukeholder**

### CODE REVIEW

**Comment by lukeholder** [comment]
Right clicking on the screen saver and clicking delete did not remove the screen saver for me. This is really needed because I had duplicate arial screen savers and didn't know which one I should delete.

**Comment by alistairmcmillan** [comment]
Good. But should probably update it to also mention the Cache folder. No point keeping 5+GB worth of MOV files if they aren't being used.

**Comment by skybleu** [comment]
In the location provided I have only symlink to brew files location. Perhaps it would be possible to uninstall directly from homebrew?

**Comment by alistairmcmillan** [comment]
@skybleu If you installed with brew, does `brew cask uninstall aerial` not work?

**Comment by skybleu** [comment]
It would work. I'm just pointing out that there's no mention of it in the proposed Uninstall section, and that might lead to removing only the alias.

File: Readme.md [hunk 1]
```diff
@@ -64,5 +64,13 @@ Issue the following terminal command: `brew cask install aerial`
 
 - Black screen: Try disabling Little Snitch, or try the latest beta release from [Aerial Releases](https://github.com/JohnCoates/Aerial/releases/).
 
+## Uninstallation
+
+Typically, you would right click on the screen saver and select 'Delete Ariel' in your system screen saver preferences. If that does not work, the screen saver can manually be deleted from the following locations:
```
Comment (Haroenv) [inline Readme.md]: Should be `Delete Aerial`

**Pull request closed without merging**